openapi/backoffice/openapi.yaml

```diff
@@ -657,6 +657,7 @@ definitions:
       - condition
       - lastUpateDate
       - state
+      - discountUrl
```

Comment:
Is it required?

Reply:
No. It's optional.

Reply:
Fixed in https://github.com/pagopa/cgn-onboarding-portal-backend/pull/117/commits/d14278fffc85d90df9671eee2b3541fcda1d7642